Formula/llvm.rb

```diff
@@ -130,6 +130,7 @@ class Llvm < Formula
   option "with-python", "Build bindings against custom Python"
   option "with-shared-libs", "Build shared instead of static libraries"
   option "without-libffi", "Do not use libffi to call external functions"
+  option "without-experimental", "Build libc++experimental library (only if libc++ build is enabled)"
```

Comment:
Good point, fixed. I actually had `with-experimental` at beginning, but decided that it does not hurt to include it by default, but forgot to change the description.
Fixed now.